### PR TITLE
Fix DisplayVersion for Google.AndroidStudio 2023.1.1.28

### PR DESCRIPTION
**Pull request by mdanish-kh**
Related to #108921

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140086)

### CODE REVIEW

File: manifests/g/Google/AndroidStudio/2023.1.1.28/Google.AndroidStudio.installer.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ InstallerSuccessCodes:
 - 1223
 UpgradeBehavior: install
 AppsAndFeaturesEntries:
-- DisplayVersion: "2023.1.1.28"
+- DisplayVersion: "2023.1.1"
```
Comment (treysis) [inline manifests/g/Google/AndroidStudio/2023.1.1.28/Google.AndroidStudio.installer.yaml:15]: Should be `2023.1`

Comment (mdanish-kh) [reply manifests/g/Google/AndroidStudio/2023.1.1.28/Google.AndroidStudio.installer.yaml:15]: Yikes, thanks for the catch

**Comment by treysis** [comment]
This is weird:
image

**Comment by mdanish-kh** [comment]
> This is weird: image

The error is actually because pipelines don't allow a duplicate DisplayVersion. This was included in another version that you removed :') Error should be gone now

**Comment by treysis** [comment]
Probably needs rebase before?

**Comment by mdanish-kh** [comment]
I re-ran the pipelines, this should fix itself. I believe the other PR needs to be `Publish-Pipeline-Succeeded` first for the check to work again, we may get the label again if that hasn't happened yet

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**